packages/core/src/libraries/logto-config.ts

```diff
@@ -160,17 +164,22 @@ export const createLogtoConfigLibrary = ({
 
     const customizerScriptsFromDatabase = getJwtCustomizerScripts(jwtCustomizers);
 
-    const newCustomizerScripts: { [key in LogtoJwtTokenKey]?: string } = {
-      [payload.key]: payload.value.script,
+    const newCustomizerScripts: CustomJwtDeployRequestBody = {
+      /**
+       * Only add `/test` endpoint for Cloudflare workers when testing.
+       * O/w overwrite the existing JWT customizer script.
+       */
+      [payload.key]: payload.isTest
```

Comment:
non-blocking: i doubt if `isTest` is good - how about we just pass a `useCase: 'test' | 'production'` and use it directly?

Reply:
Either is fine for me, but seems `useCase` could be more  straightforward. Updated.